ccd-definition/CaseEvent/CareSupervision/MultiState.json

```diff
@@ -366,7 +366,7 @@
     "PreConditionState(s)": "Open;Submitted;Gatekeeping;PREPARE_FOR_HEARING;RETURNED",
     "PostConditionState": "*",
     "SecurityClassification": "Public",
-    "ShowSummary": "N",
+    "ShowSummary": "Y",
```

Comment:
Note that this changes it for all states not just open and returned. But it is probs better this way anyway.